compiler/luci/pass/src/RemoveRedundantTransposePass.cpp

```diff
@@ -90,30 +90,26 @@ bool remove_consecutive_transpose_function(luci::CircleTranspose *target_node)
 
 namespace luci
 {
+
 /**
  *  BEFORE
  *         |
  *   [CircleNode]     [CircleConst]
- *    (main_node)      (main_perm)
- *         \               /
+ *         |           (pred_perm)
```

Comment:
https://github.com/Samsung/ONE/blob/e2014ebf11b6e43c049ad7c46485a1734513e88e/compiler/luci/pass/src/RemoveRedundantTransposePass.cpp#L48-L50
If we change this comment from `main_perm`to `pred_perm`, then this should be changed to matching with comment.